jujugui/static/gui/src/app/app.js

```diff
@@ -1110,6 +1110,7 @@ YUI.add('juju-gui', function(Y) {
         var charm = app.db.charms.getById(service.get('charm'));
         var relatableApplications = utils.getRelatableApplications(
           this.db, models.getEndpoints(service, this.endpointsController));
+        const ecs = this.env.get('ecs');
```

Comment:
I started using es6 variable definitions part way through this work. We should work towards using these so as more branches go on we'll update existing code.